src/System.Collections/ref/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
+      netstandard;
```

Comment:
@weshaggard when building the refs, do specify the actual target (like UAP, netfx..etc.) of the fallback to netstandard build will be ok? I am asking because I got many failures while building the refs for netfx because missing configuration in the failed projects. and I am wondering should I explicitly add netfx there or just netsatndard shoudl do it?